extensions/miro/src/change-role.tsx

```diff
@@ -0,0 +1,46 @@
+import { BoardMember } from "@mirohq/miro-api";
+import { Action, ActionPanel, Form, showToast, Toast, useNavigation } from "@raycast/api";
+import { capitalizeFirstLetter } from "./helpers";
+import ListMembers from "./list-member";
+import * as miro from "./oauth/miro";
+
+// change role react component
+export default function ChangeRole({
+  id,
+  memberId,
+  role,
+}: {
+  id: string;
+  memberId: string;
+  role: BoardMember["role"];
+}) {
+  const defaultRole = capitalizeFirstLetter(role ? role : BoardMember.RoleEnum.Viewer);
+  const { push } = useNavigation();
+
+  return (
+    <Form
+      actions={
+        <ActionPanel>
+          <Action.SubmitForm
+            title="Change Role"
+            onSubmit={async (value) => {
+              try {
+                await miro.changeBoardMemberRole(id, memberId, value.role);
+                await showToast({ style: Toast.Style.Success, title: "🎉 Changed role!" });
+                push(<ListMembers {...{ id }} />);
```

Comment:
you should `pop` instead of `pushing` here I think

Reply:
Ookidoki will look into it